### PR TITLE
Fix broken tests

### PR DESCRIPTION
**Pull request by ehgmsdk20**
Changes:

- Add test files for he test
- Get protoVer from blockchain
- Update snapshot

### CODE REVIEW

File: __tests__/ain_raw.test.ts [hunk 1]
```diff
@@ -364,12 +364,13 @@ describe('ain-js', function() {
...
 
     it('get with empty op_list', async function() {
+      const protoVer = (await ain.net.getProtocolVersion()).result;
       expect(await ain.db.ref(allowed_path).get([]))
       .toEqual({
         "result": null,
         "code": 30006,
         "message": "Invalid op_list given",
-        "protoVer": "1.0.7",
+        "protoVer": protoVer,
```
Comment (liayoo) [inline __tests__/ain_raw.test.ts:373]: could you use the eraseProtoVer()?

Comment (liayoo) [reply __tests__/ain_raw.test.ts:373]: cc: @platfowner

**Review by liayoo — APPROVED** [review]
LGTM with a comment, thanks!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: __tests__/ain_raw.test.ts (hunk 1 above)
Comment (ehgmsdk20) [reply __tests__/ain_raw.test.ts]: Fixed, Thanks for review!

**Pull request merged**